pkg/webhook/server.go

```diff
@@ -168,6 +174,23 @@ func (s *Server) Start(stop <-chan struct{}) error {
 		GetCertificate: certWatcher.GetCertificate,
 	}
 
+	// load CA to verify client certificate
+	if len(s.CAName) > 0 {
```

Comment:
Is there a reason you used `len(s.CAName)` over `s.CAName != ""`? Is there some optimization that I'm unaware of?

Reply:
No, just personal coding habit.. Fixed, thanks!